src/plonk/lookup/prover.rs

```diff
@@ -543,57 +543,35 @@ impl<C: CurveAffine> Constructed<C> {
 }
```

Comment:
Clippy says some struct fields are never read after this change. If that's correct, remove those fields.